lib/components/progress_bar.dart

```diff
@@ -0,0 +1,176 @@
+import 'package:flutter/material.dart';
+import 'package:monsterjump/utils/Country.dart';
+import 'package:percent_indicator/percent_indicator.dart';
+import 'package:http/http.dart' as http;
+import 'dart:convert';
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+class ProgressBar extends StatefulWidget {
+  @override
+  ProgressBarState createState() => ProgressBarState();
+}
+
+class ProgressBarState extends State<ProgressBar> {
+  Future<String> fetchData() async {
+    final response = await http.get('https://corona.lmao.ninja/countries');
```

Comment:
You can directly fetch data for Germany: https://corona.lmao.ninja/docs/#/Countries/get_countries__country_country_____

Reply:
that would be better indeed, but i dont want to spend more time on this. can we leave it as is?

Reply:
ok